src/components/Button/index.tsx

```diff
@@ -282,7 +242,8 @@ function Button({
             ref={forwardedRef}
             onPress={(event) => {
                 if (event && event.type === 'click') {
-                    event.currentTarget.blur();
+                    const currentTarget = event?.currentTarget as HTMLElement;
```

Comment:
Why we need assertion here?

Reply:
without assertion ts thinks that `currentTarget` is type of `number | EventTarget` on which `blur` function is not existing but we can assert that its `HTMLElement` because `click` event type is only on web or desktop